samcli/lib/package/artifact_exporter.py

```diff
@@ -616,6 +637,7 @@ def export(self):
             return self.template_dict
 
         self.template_dict = self.export_global_artifacts(self.template_dict)
+        self.template_dict = self.apply_global_values(self.template_dict)
```

Comment:
To be future compat, you might wanna move this before `export_global_values`

Reply:
Done